region/client.go

```diff
@@ -359,7 +359,7 @@ func (c *client) readFully(buf []byte) error {
 func (c *client) sendHello(ctype ClientType) error {
 	connHeader := &pb.ConnectionHeader{
 		UserInfo: &pb.UserInformation{
-			EffectiveUser: proto.String("gopher"),
+			EffectiveUser: proto.String("root"),
```

Comment:
An option is probably better in the long run. I will add that in a bit instead. This was mostly a quick fix as root is a more likely user than gopher ;).

Thinking about it though, it should be trivial to make this configurable with options. Let me redo this one.